app/services/cycle_timetable.rb

```diff
@@ -0,0 +1,100 @@
+class CycleTimetable
+  CYCLE_DATES = {
+    2021 => {
+      find_opens: DateTime.new(2020, 10, 6, 9, 0, 0),
+      apply_opens: DateTime.new(2020, 10, 13, 9, 0, 0),
+      deadline_banner: DateTime.new(2021, 7, 12, 9, 0, 0),
+      apply_1_deadline: DateTime.new(2021, 8, 24, 18, 0, 0),
+      apply_2_deadline: DateTime.new(2021, 9, 20, 18, 0, 0),
+      find_closes: DateTime.new(2021, 10, 3, 0, 0, 0),
+    },
+    2022 => {
+      find_opens: DateTime.new(2021, 10, 5, 9, 0, 0),
+      apply_opens: DateTime.new(2021, 10, 12, 9, 0, 0),
+      # the dates from below here are not the finalised but are required for
+      # the current implementation
+      deadline_banner: DateTime.new(2022, 7, 12, 9, 0, 0),
+      apply_1_deadline: DateTime.new(2022, 8, 24, 18, 0, 0),
+      apply_2_deadline: DateTime.new(2022, 9, 20, 18, 0, 0),
+      find_closes: DateTime.new(2022, 10, 3, 0, 0, 0),
+    },
+  }.freeze
+
+  def self.current_year
+    now = DateTime.now
+
+    CYCLE_DATES.keys.detect do |year|
+      return year if last_recruitment_cycle_year?(year)
+
+      now.between?(CYCLE_DATES[year][:find_opens], CYCLE_DATES[year + 1][:find_opens])
+    end
+  end
+
+  def self.next_year
+    current_year + 1
+  end
+
+  def self.find_closes
+    date(:find_closes)
+  end
+
+  def self.deadline_banner
+    date(:deadline_banner)
+  end
+
+  def self.apply_1_deadline
+    date(:apply_1_deadline)
+  end
+
+  def self.apply_2_deadline
+    date(:apply_2_deadline)
+  end
+
+  def self.find_opens
+    date(:find_opens, current_year)
+  end
+
+  def self.find_reopens
+    date(:find_opens, next_year)
+  end
+
+  def self.apply_reopens
+    date(:apply_opens, next_year)
+  end
+
+  def self.preview_mode?
+    Time.zone.now.between?(apply_2_deadline, find_closes)
+  end
+
+  def self.find_down?
+    return false if Time.zone.now > find_opens && Time.zone.now < find_closes
```

Comment:
This was updated in the Apply class to:
`Time.zone.now.between?(find_closes.end_of_day, find_reopens.beginning_of_day)`

Reply:
So i do that it tries to look for it in the 2023 cycle for some reason.